src/app/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,14 +1,12 @@
-<div class="container-fluid">
-	<div class="row">
-		<div class="col-12">
-			<div class="breadcrumb" *ngFor="let item of items">
-				<app-breadcrumbs-item
-					class="breadcrumb-item"
-					[title]="item.title"
-					[link]="item.link"
-					[typeOpen]="item.typeOpen"
-				></app-breadcrumbs-item>
-			</div>
+<div class="row no-gutters">
+	<div class="col-12">
+		<div class="breadcrumb" *ngFor="let item of items">
+			<app-breadcrumbs-item
+				class="breadcrumb-item"
+				[title]="item.title"
```

Comment:
It might be easier to pass the whole item into `app-breadcrumbs-item`

Reply:
fixied